### ISSUE TITLE
code question

### ISSUE THREAD

**Issue by BO41** [TeamNewPipe/website#88]
are the css files in `/css` used?
and are they minified before sending to the user?

Because `/css/blog.css` for example looks horrible and should be refactored.
Do you see this as an issue too?

- inconsistent spacing
- multiple media Queries with the same selector
- comments
- typos
- not optimized styles
- duplicate  definitions
- many floats and `!important`s
```css
border-left: 2px solid #d0cdcd;
border-left: 2px solid #f3eff2;
border-left: 2px solid white;
```

**Comment by TheAssassin** [comment]
If you feel something can be improved please feel free to send a PR.

**Comment by TobiGr** [comment]
@BO41 I wanted to use SCSS in the future. the blog stylesheet is horrible because I developed the blog in a separate repository and then merged it into this one. Given the short amount of time I had left before releasing the blog, I did this without optimizing the structures and styles.
When I made changes to styles which were not only used in the blog, but also in other parts of the project later on, I only added them. I just refactored many Jekyll/Liquid templates and includes which were messy for the same reason too. So if you are interested in improving the stylesheets, here is what I've planned for the future:

`base.scss` (this contains no styles, but all colors and mixins. It is basically the lib to import)
`main.scss` (all structures which are used across all subpages: nav, footer, tiles, etc)
Then for all the subpages, there should be single stylesheets like `home.scss`, `blog.scss`, `press.scss`, `donate.scss`, `faq.scss` which extend the two other scripts.
